docs/README.md

```diff
@@ -12,7 +12,7 @@ focus on solving mundane but difficult tasks such as:
 
 - [Deploying a container](./serving/getting-started-knative-app.md)
 - [Routing and managing traffic with blue/green deployment](./serving/samples/blue-green-deployment.md)
-- [Scaling automatically and sizing workloads based on demand](./serving/configuring-the-autoscaler.md)
+- [Scaling automatically and sizing workloads based on demand](./serving/autoscaling.md)
```

Comment:
Need to update to configuring-autoscaling.md - no need to raise an issue for this, I will do it in another open PR.